src/main/java/step2/model/WinningLotto.java

```diff
@@ -32,15 +27,13 @@ public int countOfMatch(List<Integer> lotto) {
 			.count();
 	}
 
-	public boolean isBonus(int countLottoMatch, List<Integer> lotto) {
-		long countOfBonus = DEFAULT_COUNT_OF_BONUS;
-
-		if (countLottoMatch == SECOND_MATCH_NUM) {
-			countOfBonus = lotto.stream()
-				.filter(lottoNumber -> lottoNumber.equals(bonusNumber))
-				.count();
-		}
-
-		return countOfBonus > DEFAULT_COUNT_OF_BONUS;
+	public boolean isBonus(List<Integer> lotto) {
+		return lotto.stream()
+			.filter((lottoNumber) -> lottoNumber.equals(bonusNumber))
+			.findFirst()
+			.isPresent();
```

Comment:
구매한 6개의 로또 번호들 중에서 보너스 번호가 포함됐는지를 확인하는 로직이니,
`contains()` 메서드를 활용하여 동일한 결과를 얻을 수도 있을 것 같아요.
```suggestion
		return lotto.contains(bonusNumber);
```